.changeset/shaggy-yaks-shop.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sap-cloud-sdk/generator': minor
+---
+
+[Compatibility Notes] Deprecate `writeReadme` option. You can use the new option `readme` to set if the generator will write README.md file instead.
```

Comment:
I am not sure if you need separate files for each note. @florian-richter I guess the changeset will handle multiple entries here?

Reply:
Not multiple entries per file but you can use a markdown list as you see here: https://github.com/SAP/cloud-sdk-backlog/wiki/Changesets#best-practices